types/ldapjs/index.d.ts

```diff
@@ -281,3 +292,94 @@ export class NotFilter extends Filter {
 export class ApproximateFilter extends Filter {
 	constructor(options: { attribute: string, value: string })
 }
+
+export class AttributeJson {
```

Comment:
Same here; it appears that there is no class called `AttributeJson` that's actually exposed from the library at runtime.

Reply:
Good catch, this was a mistake.